app/views/shared/proposals/_tags_form.html.haml

```diff
@@ -1,13 +1,11 @@
 %fieldset
-  = form_for proposal, url: reviewer_event_proposal_path(event, proposal), html: {role: 'form', remote: true} do |f|
-    .form-group
-      .tag-list
-        = f.label :review_tags
-        - tag_count = event.review_tags.count
-        - if tag_count > 0
+  - if event.review_tags.any?
+    = form_for proposal, url: reviewer_event_proposal_path(event, proposal), html: {role: 'form', remote: true} do |f|
+      .form-group
+        .tag-list
           = f.select :review_tags,
             options_for_select(event.review_tags, proposal.object.review_tags),
             {}, { class: 'multiselect review-tags', multiple: true }
           %button.pull-right.btn.btn-success{:type => "submit"} Update
-        - else
-          None
+  - else
+    None
```

Comment:
@gitcow the story encompasses letting organizers still ADD tags, I think. At /events/seedconf/staff/config you can add both types of tags. I don't know the exact review tags functionality.